examples/index.html

```diff
@@ -60,6 +62,20 @@
     </details-menu>
   </details>
 
+  <h1>Autofocus example</h1>
+  <p><code>summary</code> may have <code>autofocus</code> so it's the initially focused element in the page.</p>
+  <p>Then any focusable element inside the popup can declare autofocus too, so it gets focus when the popup is opened.</p>
+
+  <details>
+    <summary data-menu-button autofocus>Autofocus picker</summary>
+    <details-menu>
+      <input type="text" placeholder="Autofocus wannabe filter" autofocus />
```

Comment:
Placeholders are meant for examples of input. We can skip it here or change it to a robot name.

```suggestion
      <input type="text" autofocus />
```

```suggestion
      <input type="text" placeholder="Bender" autofocus />
```